CONTRIBUTING.md

```diff
@@ -567,3 +572,13 @@ These helper methods should generally be placed in the `at.hannibal2.skyhanni.ut
 compatability methods for. For example, `WorldClient.getAllEntities()` could be placed in `WorldCompat.kt`. This is not a strict rule, but
 it is a good guideline to follow as for the most part we do not want to be doing large amount of preprocessing in the feature files
 themselves.
+
+
+### Access wideners
+
+You may want to use private minecraft methods or fields, this is where access wideners come in. 
```

Comment:
we have mixins. whats the difference? And is this only for 1.21? if so, please mark it accordingly

Reply:
Its in the 1.21 section. its like mixins but you dont have to write a mixin or use a mixin